spec/features/view_spaces_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'capybara/rspec'
+
+feature "view spaces" do
+  scenario "user can see the available spaces" do
+    visit("/spaces")
+
+    expect(page).to have_content("Available Spaces")
```

Comment:
We should change this test to use the create method from the other branch & insert a space into the page before testing. Expecting that specific space to appear on the page. At the moment this doesn't test fcor whether it shows spaces, just the title